File: VB/RuleConversion2.Module/Updater/Registration.vb
Imports DevExpress.ExpressApp.Model
Imports DevExpress.ExpressApp.Model.Core
Imports DevExpress.ExpressApp.ConditionalFormatting
Imports DevExpress.ExpressApp.ConditionalEditorState

Namespace RuleConversion2.Module
	Partial Public Class RuleConversion2Module
		Public Overrides Sub ExtendModelInterfaces(ByVal extenders As ModelInterfaceExtenders)
			MyBase.ExtendModelInterfaces(extenders)
			extenders.Add(Of IModelColumn, IModelConditionalFormatting)()
			extenders.Add(Of IModelListView, IModelConditionalFormatting)()
			extenders.Add(Of IModelApplication, IModelApplicationConditionalEditorStates)()
		End Sub
		Public Overrides Sub AddModelNodeUpdaters(ByVal updaterRegistrator As IModelNodeUpdaterRegistrator)
			MyBase.AddModelNodeUpdaters(updaterRegistrator)
			updaterRegistrator.AddUpdater(Of IModelColumn)(New ConditionalFormattingColumnConverter())
			updaterRegistrator.AddUpdater(Of IModelListView)(New ConditionalFormattingListViewConverter())
			updaterRegistrator.AddUpdater(Of IModelConditionalEditorState)(New ConditionalEditorStateConverter())
		End Sub
		Public Overrides Sub AddGeneratorUpdaters(ByVal updaters As ModelNodesGeneratorUpdaters)
			MyBase.AddGeneratorUpdaters(updaters)
			updaters.Add(New ConditionalAppearanceNodesGeneratorUpdater())
		End Sub
	End Class
End Namespace

File: CS/RuleConversion2.Module/Updater/Registration.cs
using DevExpress.ExpressApp.Model;
using DevExpress.ExpressApp.Model.Core;
using DevExpress.ExpressApp.ConditionalFormatting;
using DevExpress.ExpressApp.ConditionalEditorState;

namespace RuleConversion2.Module {
    partial class RuleConversion2Module {
        public override void ExtendModelInterfaces(ModelInterfaceExtenders extenders) {
            base.ExtendModelInterfaces(extenders);
            extenders.Add<IModelColumn, IModelConditionalFormatting>();
            extenders.Add<IModelListView, IModelConditionalFormatting>();
            extenders.Add<IModelApplication, IModelApplicationConditionalEditorStates>();
        }
        public override void AddModelNodeUpdaters(IModelNodeUpdaterRegistrator updaterRegistrator) {
            base.AddModelNodeUpdaters(updaterRegistrator);
            updaterRegistrator.AddUpdater<IModelColumn>(new ConditionalFormattingColumnConverter());
            updaterRegistrator.AddUpdater<IModelListView>(new ConditionalFormattingListViewConverter());
            updaterRegistrator.AddUpdater<IModelConditionalEditorState>(new ConditionalEditorStateConverter());
        }
        public override void AddGeneratorUpdaters(ModelNodesGeneratorUpdaters updaters) {
            base.AddGeneratorUpdaters(updaters);
            updaters.Add(new ConditionalAppearanceNodesGeneratorUpdater());
        }
    }
}
